File: app/src/main/java/ru/buylist/data/BuyList.java
package ru.buylist.data;

public class BuyList {

    private long id;
    private String title;
    private String type;

    public BuyList() {
        id = System.currentTimeMillis();
    }

    public BuyList(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public boolean isEmpty() {
        return title == null || title.isEmpty();
    }
}

File: app/src/main/java/ru/buylist/data/entity/BuyList.kt
package ru.buylist.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "buy_lists")
data class BuyList(
        @PrimaryKey var id: Long = System.currentTimeMillis(),
        var title: String = "",
        var items: String = ""
)